tracer/test/benchmarks/Benchmarks.Trace/Asm/AppSecEncodeBenchmark.cs

```diff
@@ -21,17 +21,22 @@ namespace Benchmarks.Trace.Asm;
 [BenchmarkCategory(Constants.AppSecCategory)]
 public class AppSecEncoderBenchmark
 {
-    private static readonly Encoder _encoder;
-    private static readonly EncoderLegacy _encoderLegacy;
-    private static readonly NestedMap _args;
+    private static readonly NestedMap _args = MakeNestedMap(20);
 
-    static AppSecEncoderBenchmark()
+    private Encoder _encoder;
+    private EncoderLegacy _encoderLegacy;
```

Comment:
Out of interest, why make some of these non-static and keep others as static? 🤔 If you keep the static ones, isn't that the same issue you're trying to remove by getting rid of the static constructor?

Put another way, having a static initializer (like you have for `_args`) is the same as having a static initializer like you had before, so probably we you should either move _everything_ to non-statics, or not bother changing _anything_? 😄